src/modules/social/profiles/profiles.schema.ts

```diff
@@ -0,0 +1,73 @@
+import { z } from "zod"
+import { buildJsonSchemas } from "fastify-zod"
+import { postSchema } from "../posts/posts.schema"
+
+const profileMedia = {
+  banner: z.string().optional(),
+  avatar: z.string().optional()
+}
+
+const profileMediaSchema = z.object(profileMedia)
+
+export const profileCore = {
+  name: z.string().regex(/^[\w]+$/, "Name can only use a-Z, 0-9, and _"),
+  email: z.string({
+    invalid_type_error: "Email must be a string",
+  }).email().regex(/^[\w\-.]+@noroff.no$/, "Only noroff.no emails are allowed to register"),
```

Comment:
Need to also allow `stud.noroff.no` emails here.

I don't do regex, but here's a suggestion.
```suggestion
  }).email().regex(/^[\w\-.]+@(stud.)?noroff.no$/, "Only noroff.no emails are allowed to register"),
```

Reply:
That looks like it will work